output/components.eslint.txt

```diff
@@ -12,21 +12,24 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 
 /home/travis/build/Talend/ui/packages/components/src/Badge/Badge.component.js
-  67:58  error  Caution: `i18n` also has a named export `init`. Check if you meant to write `import {init} from 'i18next'` instead  import/no-named-as-default-member
+  67:1   error  Line 67 exceeds the maximum line length of 100                                                                                          max-len
+  67:58  error  Caution: `i18n` also has a named export `createInstance`. Check if you meant to write `import {createInstance} from 'i18next'` instead  import/no-named-as-default-member
 
 /home/travis/build/Talend/ui/packages/components/src/Enumeration/Enumeration.component.js
-  197:2   error  Missing semicolon                                                                                                   semi
-  199:58  error  Caution: `i18n` also has a named export `init`. Check if you meant to write `import {init} from 'i18next'` instead  import/no-named-as-default-member
+  197:2   error  Missing semicolon                                                                                                                       semi
+  199:1   error  Line 199 exceeds the maximum line length of 100                                                                                         max-len
+  199:58  error  Caution: `i18n` also has a named export `createInstance`. Check if you meant to write `import {createInstance} from 'i18next'` instead  import/no-named-as-default-member
 
 /home/travis/build/Talend/ui/packages/components/src/HeaderBar/__snapshots__/config.js
   1:1  error  '@storybook/react' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   4:2  error  Unexpected require()                                                                    global-require
 
 /home/travis/build/Talend/ui/packages/components/src/HeaderBar/HeaderBar.component.js
-  293:58  error  Caution: `i18n` also has a named export `init`. Check if you meant to write `import {init} from 'i18next'` instead  import/no-named-as-default-member
+  293:1   error  Line 293 exceeds the maximum line length of 100                                                                                         max-len
+  293:58  error  Caution: `i18n` also has a named export `createInstance`. Check if you meant to write `import {createInstance} from 'i18next'` instead  import/no-named-as-default-member
 
 /home/travis/build/Talend/ui/packages/components/src/List/List.component.js
-  177:58  error  Caution: `i18n` also has a named export `init`. Check if you meant to write `import {init} from 'i18next'` instead  import/no-named-as-default-member
+  177:58  error  Caution: `i18n` also has a named export `createInstance`. Check if you meant to write `import {createInstance} from 'i18next'` instead  import/no-named-as-default-member
```

Comment:
Please fix those errors

Reply:
+1
Importing createInstance won't have the same context issue than importing init.

Reply:
+1
Importing createInstance won't have the same context issue than importing init.